mapie/tests/test_metrics.py

```diff
@@ -109,3 +111,15 @@ def test_classification_y_pred_set_type() -> None:
     "Test that list(y_pred_set) gives right coverage."
     scr = classification_coverage_score(y_true_class, list(y_pred_set))
     assert scr == 0.8
+
+
+@pytest.mark.parametrize("pred_set", [y_pred_set, list(y_pred_set)])
+def test_classification_toydata_width(pred_set: ArrayLike) -> None:
+    "Test width mean for toy data."
+    assert classification_mean_width_score(pred_set) == 2.0
+
+
+def test_classification_y_pred_set_width_shape() -> None:
+    "Test shape of y_pred_set in classification_mean_width_score."
+    with pytest.raises(ValueError, match=r".*Expected 2D array*"):
+        classification_mean_width_score(y_pred_set[:, 0])
```

Comment:
What if the user chooses several values of `alpha` and `y_pred` is of shape `(n_samples, n_classes, n_alpha)` ?